src/items/items.hpp

```diff
@@ -435,6 +442,35 @@ class Items {
 		return dummys;
 	}
 
+	static const std::string getAugmentNameByType(AugmentTypes_t augmentType) {
```

Comment:
Analyze the possibility of the string being a reference to avoid unnecessary copying